server.js

```diff
@@ -0,0 +1,35 @@
+require('dotenv').config()
+const express = require('express');
+const { ApolloServer } = require('apollo-server-express');
+const typeDefs = require('./typeDefs');
+const resolvers = require('./resolvers');
+const mongoose = require('mongoose');
+const DATABASE_URL = process.env.DATABASE_URL;
```

Comment:
Think this is definitely necessary, is this all that is needed for hiding keys and things?